apps/python-sdk/firecrawl/v2/client_async.py

```diff
@@ -47,11 +47,15 @@
 from .watcher_async import AsyncWatcher
 
 class AsyncFirecrawlClient:
+    @staticmethod
+    def _is_cloud_service(url: str) -> bool:
+        return "api.firecrawl.dev" in url.lower()
```

Comment:
Can we use `process.env.CLOUD_SERVICE === 'true'` instead? Seems we already do it elsewhere?

Reply:
hmm, i don't see that env in this repo, also this is for the SDK, so we don't really want to introduce a new env var for that if we can detect it from the url